### DISCUSSION TITLE
Integrate envelop in GraphQL Mesh

### DISCUSSION THREAD

**Discussion by shadikkhan** [dotansimha/envelop#1255] [Q&A] [answered]
Is there any way to integrate envelop in GraphQL Mesh? I tried couple of ways but couldn't get through and also there is no example for this.

**Comment by dotansimha** [comment, marked as answer]
Hi @shadikkhan ! 
The latest version of Mesh supports Envelop plugins now ;) 
See: https://github.com/Urigo/graphql-mesh/pull/3627

I think it's still not fully documented, but this PR can help you to get started.
